package.json

```diff
@@ -7,7 +7,7 @@
         "bignumber.js": "2.0.0",
         "chai-as-promised": "^7.1.1",
         "openzeppelin-solidity": "^3.2.0",
-        "sinon": "^9.2.2",
```

Comment:
Removed, since the new coverage test does not work with sinon and sinon is of little value to our test actually.